erpnext_com/www/docs/user/manual/en/manufacturing/introduction.md

```diff
@@ -48,6 +48,6 @@ with the Project module.
 You can track work-in-progress by creating work-in-progress Warehouses.
 
 ERPNext will help you track material movement by automatically creating Stock
-Entries from your Work Orders by building from Bill of Materials.
+Entries from your Work Orders by creating Bill of Materials.
```

Comment:
note to self: unsure about this change, the meaning changes... need to confirm which one is right

Reply:
Ok stick to the first one... I confirmed...

Reply:
ok, so make the change